express-stock-trading-platform/server180.js

```diff
@@ -18,22 +18,22 @@ const prices = {
   OPQ: 0.48,
   RST: 9.32,
   UVW: 10.94,
-  XYZ: 5.32,
+  XYZ: 5.32
 };
 
 const checkTickerAndShares = (req, res, next) => {
   req.params.ticker = req.params.ticker.toUpperCase();
 
   if (!(req.params.ticker in prices)) {
     res.send('Error: the ticker you entered is invalid.');
-  }
-  // 180.  Now we also need to check that the number of shares that the client sent is valid, we can use the parseInt() function to do this.
-  // And an "else if" statement so that if the shares parameter cannot be parsed into an integer, you send a response saying 'Error: the number of shares submitted is invalid.'
-  else if (!parseInt(req.params.shares)) {
+  } else if (!parseInt(req.params.shares)) {
     res.send('Error: the number of shares submitted is invalid.');
   }
 };
 
+// At this point in the  code, we now know that the ticker is valid and that req.params.shares can be parsed into a valid integer.
```

Comment:
// At this point in the  code, we now know that the ticker is valid and that `req.params.shares` can be parsed into a valid integer.
// Add an `else {}` statement to the end of the conditional statement.